build/dependencies.props

```diff
@@ -4,15 +4,15 @@
   </PropertyGroup>
   <!-- AspNetCore Versions -->
   <PropertyGroup>
-    <AspNetCoreHostingVersion>1.0.2</AspNetCoreHostingVersion>
-    <KestrelVersion>1.0.3</KestrelVersion>
-    <CommandLineUtilsVersion>1.0.1</CommandLineUtilsVersion>
+    <AspNetCoreHostingVersion>1.2.0-*</AspNetCoreHostingVersion>
+    <KestrelVersion>1.2.0-*</KestrelVersion>
+    <CommandLineUtilsVersion>1.2.0-*</CommandLineUtilsVersion>
     <DotNetInternalAbstractionsVersion>1.0.0</DotNetInternalAbstractionsVersion>
-    <DependencyInjectionVersion>1.0.2</DependencyInjectionVersion>
-    <EntityFrameworkCoreVersion>1.0.3</EntityFrameworkCoreVersion>
-    <FileProvidersVersion>1.0.1</FileProvidersVersion>
-    <PlatformAbstractionsVersion>1.0.0</PlatformAbstractionsVersion>
-    <RazorVersion>1.0.2</RazorVersion>
+    <DependencyInjectionVersion>1.2.0-*</DependencyInjectionVersion>
+    <EntityFrameworkCoreVersion>1.2.0-*</EntityFrameworkCoreVersion>
+    <FileProvidersVersion>1.2.0-*</FileProvidersVersion>
+    <PlatformAbstractionsVersion>1.2.0-*</PlatformAbstractionsVersion>
+    <RazorVersion>1.2.0-*</RazorVersion>
```

Comment:
You are right. It didn't compile, so I made the changes to move to `Razor.Evolution` instead.